.github/workflows/release.yml

```diff
@@ -152,13 +152,32 @@ jobs:
             .artifacts/publish/docs-builder/release/*.zip
             .artifacts/publish/docs-assembler/release/*.zip
 
+      # Add install scripts to artifacts directory (only on Linux runner to avoid duplication)
```

Comment:
What's the reason for copying it to the `.artifacts/publish/scripts/` folder?

Reply:
Does it need to be copied anywhere?

AFAICS, you are uploading the script directly to the release.

Not sure if it's even necessary to upload it to the release.

The most common pattern I see is just sharing the raw github link.

e.g. https://raw.githubusercontent.com/elastic/docs-builder/43a5981b93cfaab58cf497ab477d14284f5c1227/install.sh

Reply:
Good point! I'll remove this release logic